app/main.go

```diff
@@ -72,6 +73,14 @@ var (
 		"stats-sd-project", "Stackdriver project for internal ts-bridge metrics",
 	).Envar("SD_PROJECT_FOR_INTERNAL_METRICS").String()
 
+	syncPeriod = kingpin.Flag(
+		"sync-period", "How often to sync metrics when running in standalone mode",
+	).Envar("SYNC_PERIOD").Default("60s").Duration()
+
+	syncCleanupAfter = kingpin.Flag(
+		"sync-cleanup-after", "Run cleanup after X sync loops",
+	).Envar("SYNC_CLEANUP_AFTER").Default("100").Int()
```

Comment:
I'd suggest making this a Duration as well.

Reply:
Duration here is not really appropriate as the amount of things to clean up is not proportional to time, rather to amount of sync loops, so I would leave it as is. Do you think that's acceptable?

Reply:
Yeah, I think I prematurely optimised (in case we needed to do some other DB maintenance).

Set up the cleanup routine to run on startup. PTAL.